doc/classes/Object.xml

```diff
@@ -469,131 +469,9 @@
-				[b]Binding and passing parameters:[/b]
-				The syntax to bind parameters is through [method Callable.bind], which returns a copy of the [Callable] with its parameters bound.
-				When calling [method emit_signal] or [method Signal.emit], the signal parameters can be also passed. The examples below show the relationship between these signal parameters and bound parameters.
-				[codeblocks]
-				[gdscript]
-				func _ready():
-				    # This assumes that a `Player` class exists, which defines a `hit` signal.
-				    var player = Player.new()
-				    # Using Callable.bind().
-				    player.hit.connect(_on_player_hit.bind("sword", 100))
-
-				    # Parameters added when emitting the signal are passed first.
-				    player.hit.emit("Dark lord", 5)
-
-				# We pass two arguments when emitting (`hit_by`, `level`),
-				# and bind two more arguments when connecting (`weapon_type`, `damage`).
-				func _on_player_hit(hit_by, level, weapon_type, damage):
-				    print("Hit by %s (level %d) with weapon %s for %d damage." % [hit_by, level, weapon_type, damage])
-				[/gdscript]
-				[csharp]
-				public override void _Ready()
-				{
-				    // This assumes that a `Player` class exists, which defines a `Hit` signal.
-				    var player = new Player();
-				    // Using lambda expressions that create a closure that captures the additional parameters.
-				    // The lambda only receives the parameters defined by the signal's delegate.
-				    player.Hit += (hitBy, level) =&gt; OnPlayerHit(hitBy, level, "sword", 100);
-
-				    // Parameters added when emitting the signal are passed first.
-				    player.EmitSignal(SignalName.Hit, "Dark lord", 5);
-				}
-
-				// We pass two arguments when emitting (`hit_by`, `level`),
-				// and bind two more arguments when connecting (`weapon_type`, `damage`).
-				private void OnPlayerHit(string hitBy, int level, string weaponType, int damage)
-				{
-				    GD.Print($"Hit by {hitBy} (level {level}) with weapon {weaponType} for {damage} damage.");
-				}
-				[/csharp]
-				[/codeblocks]
+				A signal can only be connected once to the same [Callable]. If the signal is already connected, this method returns [constant ERR_INVALID_PARAMETER] and generates an error, unless the signal is connected with [constant CONNECT_REFERENCE_COUNTED]. To prevent this, use [method is_connected] first to check for existing connections.
```

Comment:
Make the Signal class description more discoverable by adding a link to it:

```suggestion
				A [Signal] can only be connected once to the same [Callable]. If the signal is already connected, this method returns [constant ERR_INVALID_PARAMETER] and generates an error, unless the signal is connected with [constant CONNECT_REFERENCE_COUNTED]. To prevent this, use [method is_connected] first to check for existing connections.
```

Reply:
Can't do it this way, it would be very annoying to localize.
The relation to the signal class is present as a note on the bottom.